test/testdata/desugar/sclass_inheritance.rb.flattened-tree.exp

```diff
@@ -78,9 +94,19 @@ begin
     end
   end
   class <singleton class><<Class:C>> < ()
+    <emptyTree>
+
+    <emptyTree>
+
     def newerer<newerer>(<blk>)
       <self>.newer()
     end
+
+    def self.<static-init><<static-init>>(<blk>)
+      begin
+        <emptyTree>
+      end
```

Comment:
I wonder why these all show up as `begin <emptyTree> end` instead of just `<emptyTree>`.

Reply:
I think they are empty `InsSeq`s?